### PR TITLE
DA-2818: Instrument GoogleGroups Data Source

### PR DESCRIPTION
**Pull request by code-sleuth**

### CODE REVIEW

File: cmd/dads/dads.go [hunk 1]
```diff
@@ -67,6 +68,13 @@ func runDS(ctx *lib.Ctx) (err error) {
 			return err
 		}
 		return manager.Sync()
+	case googlegroups.GoogleGroups:
+		manager, err := buildGoogleGroupsManager(ctx)
+		if err != nil {
+			fmt.Println(err)
```
Comment (enkhalifapro) [inline cmd/dads/dads.go:74]: Is it for debug?

File: googlegroups/cmd.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```
Comment (enkhalifapro) [inline googlegroups/cmd.sh:1]: I think it's better to move to scripts directory

File: googlegroups/enricher.go [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+	return &enrichedMessage, nil
+}
+
+// GetEmailAddress ...
+func (e *Enricher) GetEmailAddress(rawMailString string) (email string) {
+	trimBraces := strings.Split(rawMailString, " <")
+	if len(trimBraces) > 1 {
+		email = strings.TrimSpace(trimBraces[1])
+		email = strings.TrimSpace(strings.Replace(email, ">", "", 1))
+		return
```
Comment (enkhalifapro) [inline googlegroups/enricher.go:158]: Explicit return is always better than having named output

File: googlegroups/fetcher.go [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+			break
+		}
+	}
+	if err != nil {
+		return nil, fmt.Errorf("could not read envelope: %v", err)
+	}
+	return env, nil
+}
+
+func (f *Fetcher) getMessage(archive map[string][]*enmime.Envelope, topic string, now *time.Time) (rawMessage *RawMessage, err error) {
```
Comment (enkhalifapro) [inline googlegroups/fetcher.go:190]: The same comment of named output

**Comment by lukaszgryglicki** [comment]
Is it still needed? It is a bit old...

**Comment by lukaszgryglicki** [comment]
Are we going to merge this? It looks like abandoned....

**Comment by aultron** [comment]
Closing this because we went with the gmail api implementation.

**Pull request closed without merging**